packages/website/ts/pages/governance/vote_index.tsx

```diff
@@ -0,0 +1,88 @@
+import { BigNumber } from '@0x/utils';
+import * as _ from 'lodash';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { DocumentTitle } from 'ts/components/document_title';
+import { Column, Section } from 'ts/components/newLayout';
+import { SiteWrap } from 'ts/components/siteWrap';
+import { Heading, Paragraph } from 'ts/components/text';
+import { Proposal, proposals } from 'ts/pages/governance/data';
+import { VoteIndexCard } from 'ts/pages/governance/vote_index_card';
+import { TallyInterface } from 'ts/types';
+import { documentConstants } from 'ts/utils/document_meta_constants';
+
+const ZEIP_IDS = [23, 39, 24, 25];
+const ZEIP_PROPOSALS: Proposal[] = ZEIP_IDS.map(id => proposals[id]).sort(
+    (a, b) => b.voteStartDate.unix() - a.voteStartDate.unix(),
+);
+
+export interface VoteIndexProps {}
+
+interface ZeipTallyMap {
+    [id: number]: TallyInterface;
+}
+
+export interface VoteIndexState {
+    tallys?: ZeipTallyMap;
+}
+
+export class VoteIndex extends React.Component<VoteIndexProps, VoteIndexState> {
+    public state: VoteIndexState = {
+        tallys: undefined,
+    };
+
+    public componentDidMount(): void {
+        // tslint:disable:no-floating-promises
+        this._fetchTallysAsync();
+    }
+
+    public render(): React.ReactNode {
+        return (
+            <SiteWrap>
+                <DocumentTitle {...documentConstants.VOTE} />
+                <Section isTextCentered={true} isPadded={true} padding="150px 0px 110px">
+                    <Column>
+                        <Heading size="medium" isCentered={true}>
+                            0x Protocol Governance
+                        </Heading>
+                        <Paragraph size="medium" isCentered={true} isMuted={true} marginBottom="0">
+                            Level up your favorite trading infrastructure to support new markets and industries
+                        </Paragraph>
+                    </Column>
+                </Section>
+                <VoteIndexCardWrapper>
+                    {ZEIP_PROPOSALS.map(proposal => {
+                        const tally = this.state.tallys && this.state.tallys[proposal.zeipId];
+                        return <VoteIndexCard key={proposal.zeipId} tally={tally} {...proposal} />;
+                    })}
+                </VoteIndexCardWrapper>
+            </SiteWrap>
+        );
+    }
+
+    private async _fetchTallysAsync(): Promise<void> {
+        // TODO: Real implementation
```

Comment:
FYI there is now `https://${configs.DOMAIN_VOTE}/staging` i.e `https://${configs.DOMAIN_VOTE}/staging/v1/tally/23`